06-building-a-program-start-to-finish/writing-tests/output/InvertedIndex.spec.js

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable no-undef */
+const invertedIndex = new InvertedIndex();
+const file1 = require('../books/adventureBooks.json');
+const file2 = require('../books/emptyBook.json');
+const file3 = require('../books/wrongFormat.json');
+
+const uploads = {
+  'adventureBooks.json': { content: file1 },
+  'emptyBook.json': { content: file2 },
+  'wrongFormat.json': { content: file3 }
+};
+
+describe('helpers.fileIsValid function',
+ () => {
+   let goodFile1;
+   let badFile1;
+   let badFile2;
+   let goodFile2;
+   beforeEach(() => {
+     goodFile1 = { name: 'book.json' };
+     badFile1 = { name: 'book.css' };
+     badFile2 = { name: 'book' };
+     goodFile2 = { name: 'book.JSON' };
+   });
+
+   it('should return "true" for a valid json file when its is invoked', () => {
+     expect(helpers.fileIsValid(goodFile1)).toBe(true);
+     expect(helpers.fileIsValid(goodFile2)).toBe(true);
+   });
+
+   it('should return "false" for an invalid file when it is invoked', () => {
+     expect(helpers.fileIsValid(badFile1)).toBe(false);
+     expect(helpers.fileIsValid(badFile2)).toBe(false);
+   });
+ });
+
+describe('helper.validateFileContents function',
+ () => {
+   let book3;
+   let book4;
+   beforeEach(() => {
+     book3 = { boy: 'usman', girl: 'dammy' };
+     book4 = 'this is the way we roll around here';
+   });
+
+   it('should return boolean when invoked with' +
```

Comment:
Not sure you should be concatenating here and across this entire file. Please see [Airbnb Guide](https://github.com/airbnb/javascript)